lib/hammer_cli_foreman_ansible/command_extensions/inventory.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+module HammerCLIForemanAnsible
+  module CommandExtensions
+    class Inventory < HammerCLI::CommandExtensions
+      output do |definition|
+        definition.append do
+          field :hostname, _('Name')
+          field :fqdn, _('FQDN')
+          field :hostgroup, _('Host Group')
+          field :location, _('Location')
+          field :organization, _('Organization')
+          field :domainname, _('Domain')
+          field :foreman_domain_description, _('Foreman domain description')
+          field :owner_name, _('Owner name')
+          field :owner_email, _('Owner email')
+          field :ssh_authorized_keys, _('SSH authorized keys')
+          field :root_pw, _('Root password')
+          field :foreman_config_groups, _('Foreman config groups')
+          field :puppetmaster, _('Puppet environment')
+          field :foreman_env, _('Foreman environment')
+
+          field :ansible_roles_check_mode, _('Ansible role check mode')
+          field :host_packages, _('Host packages')
+          field :host_registration_insights, _('Host registration insights')
+          field :host_registration_remote_execution, _('Host registration remote execution')
+          field :remote_execution_ssh_keys, _('Remote execution ssh keys')
+          field :remote_execution_ssh_user, _('Remote execution ssh user')
+          field :remote_execution_effective_user_method, _('Remote execution effective user method')
+          field :remote_execution_connect_by_ip, _('Remote execution connect by IP')
```

Comment:
These are not under `'foreman'` key, so if you will use `from 'foreman' do .. end`, please don't do it with these.